src/current/cockroachcloud/resource-usage.md

```diff
@@ -39,6 +41,88 @@ To understand these resources, you need to understand a bit about the CockroachD
 
 ## Diagnose and optimize your resource consumption
 
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="standard">CockroachDB Standard</button>
+  <button class="filter-button" data-scope="basic">CockroachDB Basic</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="standard">
+
+Substantial RU consumption (greater than 100 RU/second) is usually caused by SQL queries issued by the application. This can be confirmed by verifying that RU consumption tightly follows changes to the application’s SQL QPS (queries per second). On the CockroachDB {{ site.data.products.cloud }} Console [**Overview** metrics page]({% link cockroachcloud/metrics-overview.md %}), you can compare the [**Request Units** chart]({% link cockroachcloud/metrics-overview.md %}#request-units) to the [**SQL Statements** chart]({% link cockroachcloud/metrics-overview.md %}#sql-statements). Assuming the charts correlate, then reducing Request Unit consumption is about optimizing application SQL queries ([`SELECT`]({% link {{site.current_cloud_version}}/select-clause.md %}), [`UPDATE`]({% link {{site.current_cloud_version}}/update.md %}), [`INSERT`]({% link {{site.current_cloud_version}}/insert.md %}), [`DELETE`]({% link {{site.current_cloud_version}}/delete.md %})).
```

Comment:
'optimizing application SQL queries' could be a link to https://www.cockroachlabs.com/docs/stable/performance-best-practices-overview

I know it's linked below but we might as well get them the info ASAP